packages/domains-table/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { DomainsTable } from './domains-table';
 export { PrimaryDomainLabel } from './primary-domain-label';
 export { useDomainsTable } from './use-domains-table';
+export { emailManagementEdit } from './utils/paths';
```

Comment:
Looks like this wasn't supposed to be added in this PR?

Reply:
It was added intentionally, since I am using it here emailManagementEdit

Reply:
Ah, I see, it's used in another file that imports this file. 👍

Reply:
Let's remove this! See my other comment.